### ISSUE TITLE
Investigate aXe and create unit test to run in pipeline

### ISSUE THREAD

**Issue by j-tafoya** [USEPA/cam-api-portal#27]
Add to Actions pipeline

https://www.adrianbolonio.com/en/accessibility-github-actions/

**Comment by sgillEPA** [comment]
Meeting with Jonathan to set up the unit test.

**Comment by sgillEPA** [comment]
https://github.com/nickcolley/jest-axe

**Comment by sgillEPA** [comment]
//Example unit test for accessibility using jest-axe above

$ npm install jest-axe --save-dev

import React from "react"
import App from "./App"
import { render } from "@testing-library/react"
import { axe, toHaveNoViolations } from "jest-axe"

expect.extend(toHaveNoViolations)

it("should not have any accessibility violations", async () => {
  const { container } = render(<App />)
  const results = await axe(container)
  expect(results).toHaveNoViolations()
})

**Comment by sgillEPA** [comment]
https://www.youtube.com/watch?v=jC_7NnRdYb0

**Comment by sgillEPA** [comment]
Unit test was successful
